synapse/lib/webapp.py

```diff
@@ -102,14 +102,14 @@ def getFooByBar(self, bar):
         wapp.main()
 
     '''
-    def __init__(self, **settings):
+    def __init__(self, app_config={}, srv_config={}):
```

Comment:
We could also make a separate constructor function that knows how to apply app settings and server settings after construction and use that in the dmon urls?